src/index.ts

```diff
@@ -0,0 +1,40 @@
+import * as Sentry from '@sentry/node';
+import dotenv from 'dotenv';
+import yargsInit from 'yargs';
+
+import {renderServer} from './renderServer';
+import {renderStream} from './renderStream';
+import {resolveStylesConfig} from './stlyesLoader';
```

Comment:
Typo in styles.

Reply:
The real downside of typescript auto complete / imports